backend/fms_core/template_importer/importers/sample_submission.py

```diff
@@ -19,49 +20,49 @@ def import_template_inner(self):
 
         for row_id, row_data in enumerate(samples_sheet.rows):
             container = {
-                'kind': row_data['Container Kind'],
-                'name': row_data['Container Name'],
-                'barcode': row_data['Container Barcode'],
-                'coordinates': row_data['Container Coord'],
+                'kind': str_cast_and_normalize(row_data['Container Kind']),
+                'name': str_cast_and_normalize(row_data['Container Name']),
+                'barcode': str_cast_and_normalize(row_data['Container Barcode']),
+                'coordinates': str_cast_and_normalize(row_data['Container Coord']),
             }
             parent_container = {
-                'barcode': row_data['Location Barcode'],
+                'barcode': str_cast_and_normalize(row_data['Location Barcode']),
             }
             individual = {
-                'name': row_data['Individual ID'],
-                'alias': row_data['Individual Alias'],
-                'sex': row_data['Sex'],
-                'pedigree': row_data['Pedigree'],
-                'taxon': row_data['NCBI Taxon ID #'],
-                'cohort': row_data['Cohort'],
+                'name': str_cast_and_normalize(row_data['Individual ID']),
+                'alias': str_cast_and_normalize(row_data['Individual Alias']),
+                'sex': str_cast_and_normalize(row_data['Sex']),
+                'pedigree': str_cast_and_normalize(row_data['Pedigree']),
+                'taxon': int(row_data['NCBI Taxon ID #']),
```

Comment:
int(None) and int('') will throw an exception. Should the int conversion be put in a function that handles the exception and returns None?

Reply:
yeah i'll add something to the utils for that...